edb/server/server.py

```diff
@@ -557,7 +557,7 @@ async def reload_sys_config(self):
 
     def schedule_reported_config_if_needed(self, setting_name):
         setting = self._config_settings[setting_name]
-        if setting.report:
+        if setting.report and self._accept_new_tasks:
             self.create_task(
```

Comment:
Why can't `self.create_task` return early when `not self._accept_new_tasks`?

Also, `create_task(foo)` could wrap the foo coroutine into this:

```
async wrapper():
    if self._accept_new_tasks:
        await foo
```

With all that, we probably don't need to litter the code with many checks for `self._accept_new_tasks`, no?

Reply:
Unfortunately neither of the suggested methods work - if a coroutine is created but not consumed in a Task, it'll raise a warning.

Reply:
Oh, I see. :/